disnake/user.py

```diff
@@ -456,6 +468,75 @@ async def edit(
         return ClientUser(state=self._state, data=data)
 
 
+class Nameplate:
+    """
+    Represent the decoration behind the name of a user that appears
+    in the server / DM / DM group members list.
+
+    .. versionadded:: 2.11
+
+    Attributes
+    ----------
+    sku_id: :class:`int`
+        id of the nameplate SKU.
+    label: :class:`str`
+        The label of this nameplate.
+    palette: :class:`str`
+        The background color of the nameplate
+    """
+
+    __slots__ = (
+        "__state",
+        "sku_id",
+        "_asset",
+        "label",
+        "palette",
+    )
+
+    def __init__(self, state: ConnectionState, data: NameplateDataPayload) -> None:
+        self.__state: ConnectionState = state
+        self.sku_id: int = int(data["sku_id"])
+        self._asset: str = data["asset"]
+        self.label: str = data["label"]
+        self.palette = data["palette"]
+
+    @property
+    def nameplate_animated_asset(self) -> Asset:
+        """Asset: returns the animated nameplate for the user.
```

Comment:
```suggestion
        """:class:`Asset`: Returns the animated nameplate for the user.
```
Also `nameplate_static_asset`